source/server/options_impl_base.h

```diff
@@ -55,6 +55,9 @@ class OptionsImplBase : public Server::Options, protected Logger::Loggable<Logge
     parent_shutdown_time_ = parent_shutdown_time;
   }
   void setDrainStrategy(Server::DrainStrategy drain_strategy) { drain_strategy_ = drain_strategy; }
+  void setCompliancePolicy(Server::CompliancePolicy compliance_policy) {
+    compliance_policy_ = compliance_policy;
```

Comment:
This should be configuration on the tls context, not a command line option, because different listeners/clusters may need different policies.

Reply:
I think it probably belongs in `message TlsParameters` (which is a field inside `common_tls_context`).